autoload/lsp/lspserver.vim

```diff
@@ -630,6 +630,9 @@ def Rpc(lspserver: dict<any>, method: string, params: any): dict<any>
       emsg ..= $', data = {reply.error.data->string()}'
     endif
     util.ErrMsg($'Error(LSP): request {method} failed ({emsg})')
+  else
+    # No 'result'
+    HandleSymbolLocErr(lspserver, method)
```

Comment:
The RPC request can be for a method other than the symbol declaration, definition, etc.  If such a request fails, then the symbol related error message should not be printed.

Reply:
so far HandleSymbolLocErr only handle symbolloc err, such as define/declare/impls etc.
others if lsp server followed the 'error' specification, then it would print it by 'error' part.